web/src/components/apps/Dashboard.tsx

```diff
@@ -311,11 +400,13 @@ class Dashboard extends Component {
       }
     } catch (err) {
       console.log(err);
+      const errorMessage =
+        err instanceof Error
```

Comment:
`catch (err)` can always be `any` so we have to do some type narrowing checks here. I check for `instanceof Error` before trying to access `err.message`. not sure if this is the best way to do it but it seems to work.

Reply:
I've used something like this before, can we use the type Error and define that in our commonly used types interface file?
interface Error {
    name: string;
    message: string;
    stack?: string;
}

Reply:
That makes sense. I added "Add a shared Error type" to our shortcut [doc](https://app.shortcut.com/replicated/write/IkRvYyI6I3V1aWQgIjYzMjRkZDFjLTg0OWUtNDE1NS04YWMyLTU2MmU1ZjI3MWVkMyI=).